docs/contributing/Powershell Guidelines.md

```diff
@@ -0,0 +1,99 @@
+Powershell Guidelines
+==============
+
+Powershell is primarily used in this repo as the backbone of our infrastructure.  As such 
+Powershell scripts need to be portable, reliable and take advantage of stop on error 
+approaches.  The guidelines below are meant to push scripts to this mindset of execution.
+
+# Coding Style
+
+1. Opening braces always go at the end of an expression / statement. 
+1. Closing braces always occur on an otherwise empty line.
+1. We use four space indentation.
```

Comment:
I realize this is a matter of opinion, but it is fairly standard practice for shell scripts to use 2 space indentation.